threema/gateway/e2e.py

```diff
@@ -142,13 +142,13 @@ class AbstractCallback(metaclass=abc.ABCMeta):
     Raises :exc:`TypeError` in case no valid certificate has been
     provided.
     """
-    def __init__(self, connection, loop=None):
+    def __init__(self, connection, *, loop=None, route='/gateway_callback'):
```

Comment:
Hm, in theory the `*` is a breaking change. Not sure if we should drop it again, even though I think it's useful.

Reply:
I don't see why the `*` is useful.

Reply:
It forces you to name your argument, resulting in less ambiguity when reading code. It would also allow us to remove keyword arguments in the future without breaking existing codes.